manifest.json

```diff
@@ -100,6 +100,18 @@
                     "type": "string",
                     "description": "Path to the file describing your custom runtime configuration.",
                     "default": "runtime.json"
+                },
+                {
+                    "name": "useFeederCan",
+                    "type": "string",
+                    "description": "Indicates if the CAN feeder service shall be used. This service provides replay of pre-recorded CAN file.",
+                    "default": "false"
+                },
+                {
+                    "name": "useVehicleMock",
+                    "type": "string",
+                    "description": "Indicates if the vehicle mock service shall be used. This service provides a mockable interface for all VSS datapoints.",
+                    "default": "true"
                 }
```

Comment:
Wouldn't it be better - instead of this quite special solution - to extend our runtime up script with a list parameter that specifies which components to start?

Reply:
I would rather not introduce volatile parameters which need to be passed to the runtime up. By default, mock service should start - potentially we can even throw out feeder can for the default runtime completely.

Reply:
Accepted